pkg/apis/k0s/v1beta1/images.go

```diff
@@ -116,6 +116,17 @@ func (ci *ClusterImages) Validate(path *field.Path) (errs field.ErrorList) {
 		}))
 	}
 
+	errs = append(errs, ci.Konnectivity.Validate(path.Child("konnectivity"))...)
+	errs = append(errs, ci.PushGateway.Validate(path.Child("pushgateway"))...)
+	errs = append(errs, ci.MetricsServer.Validate(path.Child("metricsserver"))...)
+	errs = append(errs, ci.KubeProxy.Validate(path.Child("kubeproxy"))...)
+	errs = append(errs, ci.CoreDNS.Validate(path.Child("coredns"))...)
+	errs = append(errs, ci.Pause.Validate(path.Child("pause"))...)
+	errs = append(errs, ci.Calico.CNI.Validate(path.Child("calico").Child("cni"))...)
```

Comment:
Wondering if it's worth to move that into the CalicoImages / KubeRouterImages structs for consistency.